airflow/providers/google/cloud/transfers/sql_to_gcs.py

```diff
@@ -165,7 +168,9 @@ def _write_local_data_files(self, cursor):
             names in GCS, and values are file handles to local files that
             contain the data for the GCS objects.
         """
-        schema = list(map(lambda schema_tuple: schema_tuple[0], cursor.description))
+        org_schema = list(map(lambda schema_tuple: schema_tuple[0], cursor.description))
+        schema = [column for column in org_schema if column not in self.exclude_columns]
+
```

Comment:
Consider casting `excluded_columns` to `set` in constructor. Then you can do:
```suggestion
        org_schema = set(schema_tuple[0] for schema_tuple in cursor.description)
        schema = org_schema - self.exclude_columns

```
WDYT?

Reply:
If schema is changed to set type, the order of the columns is changed.
It seems better to use a list to keep the order of the columns.

however, it seems good to receive the input value as a set to prevent duplication of the input exclude column.
 
 
```python
            delegate_to: Optional[str] = None,
            impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
            exclude_columns=None,
            **kwargs,
    ) -> None:
        super().__init__(**kwargs)
        if exclude_columns is None:
            exclude_columns = set()
```
 using exclude_columns as a set to prevent duplication

Reply:
Sounds good to me, consider `exclude_columns = exclude_columns or set()`

Reply:
@turbaszek 
I changed the input type, please check